### ISSUE TITLE
can't access image properties

### ISSUE THREAD

**Issue by tkart38** [Mesuva/multi_file_selector_attribute#1]
hi 
is possible to access the image properties clicking on it?
i see that if i click on the last image added for one time is possible to open the dropdown of propreties but after disappear.
thanks

**Comment by Mesuva** [comment]
No, the attribute just uses one file picker to do the selecting, but then to display the list of files it just uses a simple list.

**Comment by tkart38** [comment]
Ok than for make the properties visibile i need the list of image object?
Is right?
Thanks

Il 29 Gen 2017 11:54, "Ryan Hewitt" <notifications@github.com> ha scritto:

No, the attribute just uses one file picker to do the selecting, but then
to display the list of files it just uses a simple list.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Mesuva/multi_file_selector_attribute/issues/1#issuecomment-275906567>,
or mute the thread
